paymentsheet-example/src/androidTest/java/com/stripe/android/test/core/PlaygroundTestDriver.kt

```diff
@@ -445,21 +446,64 @@ internal class PlaygroundTestDriver(
 
     fun confirmUSBankAccount(
         testParameters: TestParameters,
-        values: FieldPopulator.Values = FieldPopulator.Values(),
         afterAuthorization: (Selectors) -> Unit = {},
-        populateCustomLpmFields: FieldPopulator.() -> Unit = {},
+    ): PlaygroundState? {
+        return confirmBankAccount(
```

Comment:
Pulled the logic into `confirmBankAccount` (and `confirmBankAccountInCustomFlow`) so that ACH and ID can share most of the logic.